app/views/hostgroups/_form.html.erb

```diff
@@ -64,15 +66,17 @@
         <%= render 'puppetclasses/classes_parameters', :obj => @hostgroup  %>
       </fieldset>
       <br/>
-      <fieldset>
-        <h2><%= _('Global parameters') %></h2>
-        <% if @hostgroup.parent.present? %>
-            <h4><%= _('Parent parameters') %></h4>
-            <%= render "common_parameters/inherited_parameters", { :inherited_parameters => @hostgroup.parent_params(true), :parameters => @hostgroup.group_parameters } %>
-        <% end %>
-        <h4><%= _('Host group parameters') %></h4>
-        <%= render "common_parameters/parameters", { :f => f, :type => :group_parameters } %>
-      </fieldset>
+      <% if authorized_for(:controller => "host_editing", :action => "view_params") %>
+        <fieldset>
+          <h2><%= _('Global parameters') %></h2>
+          <% if @hostgroup.parent.present? %>
+              <h4><%= _('Parent parameters') %></h4>
+              <%= render "common_parameters/inherited_parameters", { :inherited_parameters => @hostgroup.parent_params(true), :parameters => @hostgroup.group_parameters } %>
```

Comment:
Since Parameter filters can be granular, I don't think `authorized_for` check is enough, user might be limited to just some parameters. `@hostgroup.parent_params` does not seem to solve it either. I think `.authorized(:view_params)` scope should be used at every place where Parameters are loaded. The same applies to other places of this PR.